services/image-analysis/src/main/java/services/EventController.java

```diff
@@ -0,0 +1,176 @@
+package services;
```

Comment:
We need Google copyright headers in all source files

Reply:
All done